package.json

```diff
@@ -18,19 +18,8 @@
     "check:prettier": "prettier --check .",
     "clean": "find . -name 'node_modules' -type d -prune -exec rm -rf '{}' +",
     "build": "tsc --build",
-    "localenv:network:create": "docker network create --driver=bridge --subnet=10.5.0.0/24 --gateway=10.5.0.1 rafiki",
-    "localenv:network:remove": "docker network remove rafiki",
-    "localenv:cloud-nine-wallet": "docker compose -f ./localenv/cloud-nine-wallet/docker-compose.yml",
-    "localenv:happy-life-bank": "docker compose  -f ./localenv/happy-life-bank/docker-compose.yml",
-    "localenv:tigerbeetle": "docker compose -f ./localenv/tigerbeetle/docker-compose.yml --env-file ./localenv/tigerbeetle/.env.tigerbeetle",
-    "localenv:build": "pnpm localenv:tigerbeetle build && pnpm localenv:cloud-nine-wallet build && pnpm localenv:happy-life-bank build",
```

Comment:
Can we please keep that command, obviously changing it to use the new structure

Reply:
you just use `localenv:compose build`